gneiss/_summary.py

```diff
@@ -57,3 +58,72 @@ def __init__(self, stat_results,
         # calculate the overall coefficient of determination (i.e. R2)
         sst = sse + ssr
         self.r2 = 1 - sse / sst
+
+    def _check_projection(self, project):
```

Comment:
The main reason is because a whole bunch of methods are using this check, namely `residuals` and `predict`.  Those are being added in #30